### ISSUE TITLE
Request/Proposal for Clientside Bedrock to target .NET Standard 2.0

### ISSUE THREAD

**Issue by Ninds** [davidfowl/BedrockFramework#89]
For me the use case of Bedrock is to implement a server with some bespoke protocols. Essentially an ASP.NET Core for protocol X rather than HTTP.
This I am keen to do against .NET CORE 3.X.

However, since I have non-standard, inhouse protocols, I also have to make available  clients. The constraint on clients to be build against .NET CORE is not feasible. An example of a client would be an Excel XLL, or some other older software build against .NET Framework 4.X.

Is this a reasonable request  ? If so will it be considered.

As of right now I am using MG's  "not so Unofficial"  Unofficial library.  
ConnectionContext is available in a .NET Standard 2.0 compatible Nuget package. I did have to copy over DuplexPipe from Bedrock and inline it into my Clientside library.

**Comment by davidfowl** [comment]
Yes I plan to fix the TFMs as part of the MVP. https://github.com/davidfowl/BedrockFramework/issues/49
